src/main/java/Cell.java

```diff
@@ -0,0 +1,30 @@
+import org.jetbrains.annotations.NotNull;
+
+public class Cell {
+
+    public enum Status {FREE, OCCUPIED}
+
+    private Stone stone;
+    private Status status = Status.FREE;
```

Comment:
Instead of Status, what about a boolean occupied field?

Reply:
Yes it can be